src/main/scala/firrtl/transforms/analysis/Netlist.scala

```diff
@@ -1,12 +1,11 @@
-package firrtl.analyses
+package firrtl.transforms.analysis
```

Comment:
I'm not sure that analyses should be a sub package of transform. They are pretty different imo. @albert-magyar